src/scripts/components/main/copy-to-clipboard-button/copy-to-clipboard-button.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {FunctionComponent, useEffect, useState} from 'react';
+
+import Button from '../button/button';
+import {CopyTextIcon} from '../icons/copy-text-icon';
+import {CheckIcon} from '../icons/check-icon';
+
+import styles from './copy-to-clipboard-button.module.styl';
+
+interface Props {
+  label: string;
+  handleCopy: () => void;
+}
+const CopyToClipboardButton: FunctionComponent<Props> = ({
+  label,
+  handleCopy
+}) => {
+  const [copied, setCopied] = useState(false);
+
+  useEffect(() => {
+    const timeout = setTimeout(() => {
```

Comment:
Only set the timer when copied is true?

Reply:
yes, so that the checkmark only appears for a bit:
https://github.com/ubilabs/esa-climate-from-space/assets/45818654/4bf7b429-d009-4179-b235-a3d3df8a9fbd